kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -0,0 +1,533 @@
+                  <KIcon class="reduce-chervon-spacing" icon="chevronUp" />
+                </div>
+              </KGridItem>
+
+              <KGridItem :layout12="{ span: 4 }">
+                <KIcon
+                  class="icon-size"
+                  icon="refresh"
+                />
+              </KGridItem>
+
+              <KGridItem :layout12="{ span: 4 }">
+                <KIcon
+                  class="icon-size"
+                  icon="trash"
+                />
+              </KGridItem>
+            </KGrid>
+          </div>
+        </KGridItem>
+
+      </KGrid>
+
+      <AccordionContainer
+        :items="placeholderList"
+      >
+        <template
+          #default="{ isItemExpanded, toggleItemState, isOptionSelected, isAnswerSelected }"
+        >
+          <div
+            v-for="item in placeholderList"
+            :key="item.id"
+          >
+            <AccordionItem
+              :id="item.id"
+              :key="item.id"
+              :items="placeholderList"
+              :title="item.title"
+              :expanded="isItemExpanded(item.id)"
+              @click="toggleItemID(item.id)"
+            >
+              <template
+                #heading="{ title }"
+                :accordionToggle="onAccordionToggle(item.id)"
+              >
+                <KGrid class="question-row">
+                  <KGridItem :layout12="{ span: 6 }">
+                    <div class="left-column-alignment-style">
+                      <DragHandle>
+                        <!-- FIXME: Needs vertically centered -->
+                        <KIcon
+                          class="drag-icon icon-size"
+                          icon="dragVertical"
+                        />
+                      </DragHandle>
+
+                      <div>
+                        <p>
+                          <KCheckbox />
+                        </p>
+                      </div>
+
+                      <div>
+                        <p>
+                          {{ title }}
+                        </p>
+                      </div>
+                    </div>
+                  </KGridItem>
+
+                  <KGridItem :layout12="{ span: 6 }">
+                    <div class="right-alignment-style">
+                      <KIconButton
+                        v-if="isItemExpanded(item.id)"
+                        class="icon-size"
+                        icon="chevronUp"
+                        @click="toggleItemState(item.id)"
+                      />
+
+                      <KIconButton
+                        v-else
+                        class="icon-size"
+                        icon="chevronRight"
+                        @click="toggleItemState(item.id)"
+                      />
+                    </div>
+                  </KGridItem>
+                </KGrid>
+              </template>
+
+              <template
+                v-if="isItemExpanded(item.id)"
+                #content=""
```

Comment:
No need for the `=""` here - this is just to indicate what slot the wrapped content will go into in the parent component